app/experimenter/experiments/models/nimbus.py

```diff
@@ -122,8 +122,9 @@ def targeting(self):
         if self.is_desktop_experiment:
             expressions.append("'app.shield.optoutstudies.enabled'|preferenceValue")
 
+        #  If there is no targeting defined all clients should match, so we return "true"
         if len(expressions) == 0:
-            return None
+            return "true"
```

Comment:
Will this ultimately display the word "true" in the UI?

Re: https://experimenter.services.mozilla.com/nimbus/onoff-aboutwelcome-experiment/,
It looks like for each branch's `Enabled` field, we display "True" or "False" (initial uppercase character).

Reply:
IMO it should show the exact JEXL, which should look like 
```
true
```